### DISCUSSION TITLE
How do I avoid overwrite each sounds files?

### DISCUSSION THREAD

**Discussion by RonaldMacdonaldTrump** [gildor2/UEViewer#261] [Q&A]
So, I'm exporting sound assets from Chivalry: Medieval Warfare, and there's same named(predicted) sound files, so when I saw in statics there was 1,706 files detected, but after I export sounds, It's only have 1,606 files.
![image](https://user-images.githubusercontent.com/35054761/151518352-fd6e2d6a-a8b0-4e6b-9c2f-d188ef3642c0.png)
![image](https://user-images.githubusercontent.com/35054761/151518519-68ce0e70-86ab-43a5-bc64-fd3709846af2.png)

**Comment by gildor2** [comment]
Probably you're not using "groups" and "uncook" options. I think everything's placed into a single ":SoundNodeWave" folder, right?
